src/Calculator.Shared/Views/MainPage.xaml

```diff
@@ -12,6 +12,7 @@
 			x:Name="pageRoot"
 			Background="{ThemeResource AppChromeAcrylicHostBackdropMediumLowBrush}"
 			Loaded="OnPageLoaded"
+			xmlns:android="http://nventive.com/android"
```

Comment:
don't write nventive, we have no standard right now but something like 
```suggestion
			xmlns:android="http://platform.uno/android"
```
maybe?